dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/GetServiceTaskTest.java

```
@@ -43,22 +57,54 @@
+    private static final BuildInfoServiceImpl buildInfoService = new BuildInfoServiceImpl();
+    private static final String TEST_ENV_NAME = "GetServiceTaskTest";
+    private static final String bucketName = "GetServiceTaskTest_bucket";
+    private static TempStorageIds envStorageIds;
+    private static UUID envDataPolicyId;
 
     @Before
     public void setUp() throws Exception {
+        final ResourceBundle resourceBundle = ResourceBundle.getBundle("lang", new Locale(ConfigProperties.getInstance().getLanguage()));
         new JFXPanel();
         Platform.runLater(() -> {
-            final SavedSession savedSession = new SavedSession(SessionConstants.SESSION_NAME, SessionConstants.SESSION_PATH, SessionConstants.PORT_NO,
-                    null, new SavedCredentials(SessionConstants.ACCESS_ID, SessionConstants.SECRET_KEY), false);
-            session = new CreateConnectionTask().createConnection(SessionModelService.setSessionModel(savedSession, false));
+            final SavedSession savedSession = new SavedSession(
+                    TEST_ENV_NAME,
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(
+                            client.getConnectionDetails().getCredentials().getClientId(),
+                            client.getConnectionDetails().getCredentials().getKey()),
+                    false,
+                    false);
+            session = new CreateConnectionTask().createConnection(SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+
+            try {
+                envDataPolicyId = IntegrationHelpers.setupDataPolicy(TEST_ENV_NAME, false, ChecksumType.Type.MD5, client);
+                envStorageIds = IntegrationHelpers.setup(TEST_ENV_NAME, envDataPolicyId, client);
+            } catch (IOException e) {
+                e.printStackTrace();
```

If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.